prow/plugins/trigger/pr.go

```diff
@@ -110,6 +111,14 @@ func trustedPullRequest(ghc githubClient, pr github.PullRequest, trustedOrg stri
 	} else if orgMember {
 		return true, nil
 	}
+	labels, err := ghc.GetIssueLabels(pr.Base.Repo.Owner.Login, pr.Base.Repo.Name, pr.Number)
+	if err != nil {
+		return false, err
+	}
+	// If the PR's "needs-ok-to-test" has been removed, it's trusted.
+	if !hasLabel(needsOkToTest, labels) {
+		return true, nil
+	}
 	// Next look for "/ok-to-test" comments on the PR.
```

Comment:
If we depend on the existence of `needs-ok-to-test`, seems like we don't need to check `/ok-to-test` any more. Any ideas?

Reply:
No new user will be able to add the label automatically so they will still need the comment from an authorized member.

Reply:
Yeah we still need the comment `/ok-to-test`. What I was thinking of was that this func doesn't need to look for `/ok-to-test` comments on PR if the PR doesn't have `needs-ok-to-test` label (implicitly trusted). OTOH, `needs-ok-to-test` is removed either by comment from a member (in trigger/ic.go) or by a member (maintainer) manually. In both the cases, we can trust the PR.